h5c/vic/src/vic-webapp/src/app/container-view/container-view.template.html

```diff
@@ -1,4 +1,30 @@
-<clr-datagrid (clrDgRefresh)="refreshGrid($event)" [clrDgLoading]="isDgLoading">
+<div class="col-xs-12 mb-2" *ngIf="errorObj">
+    <div class="row">
+      <div class="col-xs-12">
+        <clr-alert [clrAlertType]="'danger'"
+                  [clrAlertClosable]="false">
+          <clr-alert-item *ngIf="errorObj.type === 'ssl_cert'">
+            <span class="alert-text">
+              <!-- TODO: i18n -->
+              Failed to verify the VIC machine server at endpoint https://{{ errorObj.payload }}:8443. Your browser may be blocking the connection because of the untrusted SSL certificate.
+            </span>
+            <div class="alert-actions">
+              <a href="https://{{ errorObj.payload }}:8443/container/version" target="_blank">Fix</a>
```

Comment:
I posted a small PR that might provide a better user experience: vmware/vic#6832. Let me know what you think.

Reply:
Also, can we provide a better label than "Fix"? That implies that just clicking the button will fix the issue.

I think it's necessary to explain what we're asking the user to do and why.

Reply:
Yeah, it could be better. Apparently I was using a placeholder value